fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/makers/CountMaker.java

```diff
@@ -2,22 +2,20 @@
 // Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
 package com.yahoo.bard.webservice.data.config.metric.makers;
 
+import static java.util.Collections.emptySet;
```

Comment:
Why the static import? A static import makes sense to me if the class name is really long and we're using a function/constant a lot, but `Collections` isn't that long, and `emptySet` is only used once. So I would argue that the readability benefits of `Collections.emptySet()` outweighs the benefits from reducing verbosity.